helm/wrongsecrets-ctf-party/templates/wrongsecrets-balancer/deployment.yaml

```diff
@@ -42,30 +40,10 @@ spec:
               path: /balancer/
               port: http
           env:
-            - name: REACT_APP_MOVING_GIF_LOGO
-              value: {{ .Values.balancer.env.REACT_APP_MOVING_GIF_LOGO }}
-            - name: REACT_APP_ACCESS_PASSWORD
-              value: {{ .Values.balancer.env.REACT_APP_ACCESS_PASSWORD }}
-            - name: REACT_APP_HEROKU_WRONGSECRETS_URL
-              value: {{ .Values.balancer.env.REACT_APP_HEROKU_WRONGSECRETS_URL }}
-            - name: REACT_APP_CTFD_URL
-              value: {{ .Values.balancer.env.REACT_APP_CTFD_URL }}
-            - name: REACT_APP_S3_BUCKET_URL
-              value: {{ .Values.balancer.env.REACT_APP_S3_BUCKET_URL }}
-            - name: K8S_ENV
-              value: {{ .Values.balancer.env.K8S_ENV }}
-            - name: IRSA_ROLE
-              value: {{ .Values.balancer.env.IRSA_ROLE }} #REPLACE WITH THE ACTUAL AWS ROLE IF IN AWS MODE
-            - name: WRONGSECRETS_TAG
-              value: {{ .Values.wrongsecrets.tag}}
-            - name: WRONGSECRETS_DESKTOP_TAG
-              value: {{ .Values.virtualdesktop.tag }}
-            - name: REACT_APP_CREATE_TEAM_HMAC_KEY
-              value: hardcodedkey
-            - name: SECRETS_MANAGER_SECRET_ID_1
-              value: {{ .Values.balancer.env.SECRETS_MANAGER_SECRET_ID_1 }}
-            - name: SECRETS_MANAGER_SECRET_ID_2
-              value: {{ .Values.balancer.env.SECRETS_MANAGER_SECRET_ID_2 }}
```

Comment:
I think these need to be at deployment level unless it is shared by the namespaced objects?

Reply:
It is actually present in the generated manifests here:
https://github.com/OWASP/wrongsecrets-ctf-party/pull/249/files#diff-0e79142774e4871080c08cade4299253e8b84c26fab30421555d6a454406abf7R262

It's taken directly from the `values.yaml` by looping over the `env` map and listing all names with values. This also results in naming the env with the same name in the `values.yaml`